### ISSUE TITLE
Request: ARD and live option

### ISSUE THREAD

**Issue by clustermaster99** [thorbenprimke/ZDFMediathekRoku#4]

**Comment by thorbenprimke** [comment]
hi @clustermaster99 - I haven't worked on this app in quite a while. The app still works for me but I don't use it much anymore. My TV (Beoplay V1) now includes an app for ZDF and ARD. I also noticed that they must have added a new stream format/quality since the TV app's video quality is higher than what the app on the Roku delivers. 2 years ago when I worked on it, I picked the highest quality stream out of their XML/JSON data that they use for the website. 
If you want to work on this, I'm happy to merge in any PRs.
